src/ProjectSystemSetup/ProjectSystemSetup.csproj

```diff
@@ -88,11 +89,13 @@
   <ItemGroup>
     <None Include="project.json" />
     <None Include="source.extension.vsixmanifest">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Do not Copy</CopyToOutputDirectory>
```

Comment:
That value looks like the localized display name, not the actual value. Just remove the metadata, which defaults to DoNotCopy.